### ISSUE TITLE
Crash: Input audio data is not block aligned

### ISSUE THREAD

**Issue by stianjensen** [Kitt-AI/snowboy#193]
When using the node.js library, version 1.2.0 detection works nicely when detecting from a stream that comes from a file on disk.

But when streaming audio over the network and piping that to the detector, it crashes after a while with the following message:

```

**Comment by chenguoguo** [comment]
Is it possible that you have some missing bits when streaming the audio?

**Comment by chenguoguo** [comment]
Did you make it work? @stianjensen

**Comment by stianjensen** [comment]
I'll need to do a bit more testing when I'm back in the office over the weekend. I'll report back!

**Comment by abramovi** [comment]
Any update on this one? @stianjensen did you manage to do 
block alignment

**Comment by stianjensen** [comment]
Sorry, I never managed to get around to verify this, and I'm not working on that project anymore (don't have access to the code anymore, unfortunately).

But after my first attempt at testing chunking the data I couldn't immediately reproduce the bug, at least.

**Comment by abramovi** [comment]
@stianjensen 
Thank you, I get the same error when I stream audio using websocket  and than pipe it to the detector .
how did you thought to do block alignment?

**Comment by stianjensen** [comment]
Yeah that's the thing, I don't remember. I might have been using stream-throttle from npm. Although thinking back I'm not quite sure if that package did block alignment or not. As I said, I have lost access to the code since then so I can't go back and look.

**Comment by Loganrose** [comment]
Im having the same issue. If i use something like block-stream then the hotword triggers on everything.
